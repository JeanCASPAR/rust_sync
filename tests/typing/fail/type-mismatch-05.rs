use rust_sync::sync;

sync! {
    #![pass(1)]
    
    node oui() = ()
    where
        a : int = 3 + false;
}

fn main() {}
