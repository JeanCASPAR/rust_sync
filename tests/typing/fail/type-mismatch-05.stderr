error: type mismatch: left has type `int`, right has type `bool`
 --> tests/typing/fail/type-mismatch-05.rs:8:19
  |
8 |         a : int = 3 + false;
  |                   ^^^^^^^^^
